app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -22,6 +22,10 @@ def available_translations
 
     delegate :name, :name_full, to: :institution, prefix: :institution
 
+    def registration_available?
+      Devise.mappings[:user].registerable?
```

Comment:
Should we test for the presence of devise?  What if someone uses omniauth (without devise)?